buildSrc/src/main/groovy/TestTools.groovy

```diff
@@ -52,7 +58,12 @@ By default only runs in CI; to run locally:
             dependsOn project.tasks.findByName('testClasses')
 
             if (parallel) {
-                maxParallelForks = 12
+                if (project.hasProperty('maxParallelForks'))
+                    maxParallelForks = project.maxParallelForks as int
+                else
+                    maxParallelForks = 12
+                if (project.hasProperty('forkEvery'))
+                    forkEvery = project.forkEvery as int
             } else {
```

Comment:
It would be nice to have spotless check our groovy files - in the meantime, two things:

1. Can we have if body brackets `{}`?
2. It's better practice IMO to do `project.property('property-name')` instead of trying to directly reference it.